recipe/meta.yaml

```diff
@@ -44,7 +50,10 @@ requirements:
     - matplotlib-base
     - scipy
     - bitstring >=3.1.9,<4
-    - importlib_metadata                  # [py<310]
+    - libstdcxx-ng                        # [linux]
+    - libgcc-ng                           # [linux]
+    # Due to rocksDB: https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
```

Comment:
`rocksdb-feedstock` is missing this one, but this is suggested in the conda-forge docs